packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/ProviderVirtualMachinesRow.tsx

```diff
@@ -25,12 +25,17 @@ export const ProviderVirtualMachinesRow: React.FC<RowProps<VmData>> = ({
   );
 };
 
-const cellRenderers: Record<string, React.FC<VMCellProps>> = {
+export const defaultCellRenderers: Record<string, React.FC<VMCellProps>> = {
```

Comment:
please move content to individual providers, each provider should be independent